docker/build.sh

```diff
@@ -180,7 +180,7 @@ function build() {
 
     # Scan the image with trivy and output to stdout
     epoch=$(date +%s%N)
-    trivy -f json -o "${epoch}.json" --no-progress --exit-code 0 --severity HIGH --ignore-unfixed -timeout 1m "${latest_image}" || true
+    trivy -f json -o "${epoch}.json" --no-progress --exit-code 0 --severity HIGH --ignore-unfixed -timeout 5m -d "${latest_image}" || echo "ignoring trivy failure"
```

Comment:
Do we really want debug mode always on? I think the build logs are already hard to read...
```suggestion
    trivy -f json -o "${epoch}.json" --no-progress --exit-code 0 --severity HIGH --ignore-unfixed -timeout 5m "${latest_image}" || echo "ignoring trivy failure"
```

Reply:
I thought it would help identify the failure, since when it times out we just get an empty json. I can remove it if we don't care, but then why run this command at all?

Reply:
@danthegoodman1 @stathismor have you checked how often Trivy fails? How far are we actually to enforce Trivy?

Reply:
I would not use `-d`, unless there is a good reason. Logs are  already hard to navigate, and when trivy fails, so far, it does produce some meaningful error message I believe, definitely the timeout one.

Regarding the 5 minutes timeout, unfortunately this is not going to work for us. Not without digging in concourse (or wherever else the general build timeout is coming from) and finding how to increase a build's timeout. 5 mins is the default trivy timeout, and has caused issues in the past. Repos like Jellyfish produce lots of images, and on a bad trivy day, 5mins add up. I actually had to decrease [from 3 to 1](https://github.com/product-os/scripts/pull/148/files) to make sure we're under the limit.

Reply:
I was also going to ask on FD, I think I got distracted, where are we with this. We have definitely gathered enough data and should be able to start telling people what to fix now. We can resume this discussion on FD.